index.html

```diff
@@ -86,12 +86,17 @@ <h2 id="introduction">Introduction</h2>
     <li>
         integrates well with <a href="#tutorial-5a">autotools</a>,
         <a href="#tutorial-5b">cmake</a>, <a href="#tutorial-5c">qmake</a>,
-        and <a href="#tutorial-5d">hand-written makefiles</a>.
+        and <a href="#tutorial-5d">hand-written makefiles</a>
     </li>
     <li>
         has been in continuous development <a href="#history">since 2007</a>
         and is <a href="#used-by">used by several projects</a>
     </li>
+    <li>
+        due to the presence of <a href="http://pkg.mxe.cc/">binary
+        distribution</a>, can be used in Continuous Integration systems
+        (<a href="https://github.com/zer0main/battleship">example</a>).
+    </li>
```

Comment:
has <a href="http://pkg.mxe.cc/">pre-compiled binaries</a> that <a href="https://github.com/zer0main/battleship">can be used in Continuous Integration systems</a>